test/e2e/benchmark/throughput.go

```diff
@@ -1,26 +1,65 @@
 package main
 
 import (
-	"context"
-	"fmt"
 	"log"
 	"time"
 
 	"github.com/celestiaorg/celestia-app/v2/pkg/appconsts"
 	"github.com/celestiaorg/celestia-app/v2/test/e2e/testnet"
-	"github.com/celestiaorg/celestia-app/v2/test/util/testnode"
-	"github.com/tendermint/tendermint/pkg/trace"
 )
 
 const (
 	seed = 42
 )
 
+var bigBlockManifest = Manifest{
+	ChainID:    "test",
+	Validators: 2,
+	TxClients:  2,
+	ValidatorResource: testnet.Resources{
+		MemoryRequest: "12Gi",
+		MemoryLimit:   "12Gi",
+		CPU:           "8",
+		Volume:        "20Gi",
+	},
+	TxClientsResource: testnet.Resources{
+		MemoryRequest: "1Gi",
+		MemoryLimit:   "3Gi",
+		CPU:           "2",
+		Volume:        "1Gi",
+	},
+	SelfDelegation: 10000000,
+	// @TODO Update the CelestiaAppVersion and  TxClientVersion to the latest
+	// version of the main branch once the PR#3261 is merged by addressing this
+	// issue https://github.com/celestiaorg/celestia-app/issues/3603.
+	CelestiaAppVersion: "pr-3261",
+	TxClientVersion:    "pr-3261",
+	EnableLatency:      false,
+	LatencyParams:      LatencyParams{70, 0}, // in  milliseconds
+	BlobSequences:      60,
+	BlobsPerSeq:        6,
+	BlobSizes:          "200000",
+	PerPeerBandwidth:   5 * testnet.MB,
+	UpgradeHeight:      0,
+	TimeoutCommit:      11 * time.Second,
+	TimeoutPropose:     80 * time.Second,
+	Mempool:            "v1", // ineffective as it always defaults to v1
+	BroadcastTxs:       true,
+	Prometheus:         false,
+	GovMaxSquareSize:   512,
+	MaxBlockBytes:      7800000,
+	TestDuration:       5 * time.Minute,
+	LocalTracingType:   "local",
+	PushTrace:          true,
+}
+
 func TwoNodeSimple(logger *log.Logger) error {
 	latestVersion, err := testnet.GetLatestVersion()
 	testnet.NoError("failed to get latest version", err)
 
-	logger.Println("=== RUN TwoNodeSimple", "version:", latestVersion)
+	testName := "TwoNodeSimple"
+	logger.Printf("Running %s\n", testName)
+	logger.Println("version", latestVersion)
```

Comment:
is this missing a formatting directive?

```suggestion
	logger.Printf("version %v\n", latestVersion)
```

Reply:
They both indeed work identically:
The `Println` function prints the supplied inputs with comma separation, whereas in `Printf`, I achieved the same result using formatting. Here is a sample output from the tests:
```
test-e2e-benchmark2024/07/08 11:34:05 Running TwoNodeSimple
test-e2e-benchmark2024/07/08 11:34:05 version 8caa580
```
Their implementation is just inconsistent, which I can fix in a follow-up PR.

Reply:
Please see https://github.com/celestiaorg/celestia-app/pull/3670